lib/gruf/client/error.rb

```diff
@@ -50,17 +50,19 @@ class InvalidArgument < Validation; end
       class NotFound < Validation; end
       class AlreadyExists < Validation; end
       class OutOfRange < Validation; end
-
+      
```

Comment:
Can you remove the empty tab space here?

Reply:
sorry, just realize how unreliable is the web editor of git. Doing it in my IDE.